recipes/dakota/meta.yaml

```diff
@@ -0,0 +1,77 @@
+{% set name = "dakota" %}
+{% set version = "6.10" %}
+{% set sha256sum = "d989f77577199497dfcd49e4b13b85cb7dbe20e82fc28a15b4d3811d9e362f21" %}
+
+package:
+    name: {{ name }}
+    version: {{ version }}
+
+source:
+    fn: dakota{{ version }}.tar.gz
+    url: https://dakota.sandia.gov/sites/default/files/distributions/public/dakota-{{ version }}-release-public.src.tar.gz
+    sha256: {{ sha256sum }}
+    patches:
+        - surfpack_py_bindings.patch
+        - gsl.patch
+        - python.patch
+        - colin_boost170.patch
+
+build:
+    number: 0
+    skip: true  # [win or osx or py<37]
+
+requirements:
+    build:
+        - cmake
+        - {{ compiler('c') }}
+        - {{ compiler('cxx') }}
+        - {{ compiler('fortran') }}
+    host:
+        - openmpi
+        - pybind11
+        - boost-cpp
+        - libblas
+        - libcblas
+        - python
+        - perl
+        - numpy
+        - swig
+        - doxygen
+        - gsl
+        # - trilinos  # needs newer trilinos
+    run:
+        - openmpi
+        - boost-cpp
+        - python
+        - numpy
```

Comment:
I think this should be `pin_compatible("numpy")`: https://conda-forge.org/docs/maintainer/adding_pkgs.html#jinja-templating